osdep/mpv.rc

```diff
@@ -16,6 +16,7 @@
  */
 
 #include <winver.h>
+#include "version.h"
```

Comment:
Re-adding "version.h" to mpc.rc brings back the build failures from 3375d65bc5c1. For some reason we couldn't figure out how make waf wait for version.h to be generated before compiling mpc.rc.

If we can figure out how to fix this, we could also go back to having real version numbers in mpc.rc.